build/configs/rtl8720e/scripts/rlx8720e_img2_ns.ld

```diff
@@ -83,7 +83,7 @@ MEMORY
 	/* IMG2 RAM */
 	KM4_BD_RAM (rwx) :                      ORIGIN = 0x20012020, LENGTH = 0x20040000 - 0x20012020	/* KM4 BD RAM: 184k */
 	KR4_BD_RAM (rwx) :                      ORIGIN = 0x20040060, LENGTH = 0x2007A000 - 0x20040060	/* KR4 BD RAM: 232k */
-	KM4_HEAP_EXT (rwx) :                    ORIGIN = 0x2007A000, LENGTH = 0x200B0000 - 0x2007A000	/* KM4 HEAP EXT: 280k(8720EA) */
+	KM4_HEAP_EXT (rwx) :                    ORIGIN = 0x2007A000, LENGTH = 0x200B0000 - 0x2007A000	/* KM4 HEAP EXT: 216k(8720EA) */
```

Comment:
Do we not need to update the script for trustzone disabled scenario <rlx8720e_img2_is.ld> ?